src/v2/imports/carts/buddi.ts

```diff
@@ -1,90 +1,57 @@
-import { errorTrackingSource } from "../../../shared/sources/error-tracking-source";
-import { xhrResponseSource } from "../../../shared/sources/xhr-response-source";
 import { QueryStringContext } from "../../../shared/types";
+import buddiTrackerImport from "../../../shared/environment-data-sources/buddi";
 
 const buddiTracker = ({ appId, retailId }: Pick<QueryStringContext, "appId" | "retailId">): void => {
-  xhrResponseSource((xhr: XMLHttpRequest): void => {
-    errorTrackingSource(() => {
-      const response = JSON.parse(xhr.responseText);
-      const cartList = []; // Creates an empty array to store cart items for comparison with removeFromCart array
-
-      if (xhr.responseURL.includes("cart") && !xhr.response.includes("delete")) {
-        const product = response;
-
-        cartList.push(product);
-
-        window.tracker("trackAddToCart", {
-          sku: product.id.toString(),
-          name: product.name.toString() || "N/A",
-          category: "N/A",
-          unitPrice: parseFloat(product.price) || 0,
-          quantity: parseInt(product.qty) || 1,
-          currency: "USD",
+  buddiTrackerImport({
```

Comment:
Rename this to `buddiDataSource`

Reply:
Done, Thanks for the feedback @pacholoamit